tests/gplates_1_3/screen-output

```diff
@@ -38,7 +31,7 @@ Number of degrees of freedom: 10,656 (6,528+864+3,264)
    Solving Stokes system... 30+16 iterations.
 
    Postprocessing:
-     RMS, max velocity:                  0.0256 m/year, 0.102 m/year
+     RMS, max velocity:                  0.0257 m/year, 0.0769 m/year
```

Comment:
You're on a very coarse mesh, so the difference in interpolation may well make the difference. It would be worth seeing whether you get smaller differences from this testcase if you run it with a once-refined mesh with and without your patches.